### PR TITLE
Clean up docker integration test and add documentation

### PR DESCRIPTION
**Pull request by shamrickus**
## What does this PR (Pull Request) do?

- [x] This PR is not related to any issue.

This cleans up the dockerfile for the TP integration test, as well as update the documentation with how to run the TO/TP integration tests. It also ignores a folder created for the docker integration tests.

## Which Traffic Control components are affected by this PR?

- CDN in a Box
- Documentation

## What is the best way to verify this PR?
Build docs, follow instructions to run Docker integration containers.

## The following criteria are ALL met by this PR

- [x] This PR includes tests OR I have explained why tests are unnecessary
- [x] This PR includes documentation OR I have explained why documentation is unnecessary
- [x] This PR includes an update to CHANGELOG.md OR such an update is not necessary
- [x] This PR includes any and all required license headers
- [x] This PR ensures that database migration sequence is correct OR this PR does not include a database migration
- [x] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY** (see [the Apache Software Foundation's security guidelines](https://www.apache.org/security/) for details)

### CODE REVIEW

File: docs/source/admin/quick_howto/ciab.rst [hunk 1]
```diff
@@ -104,6 +104,21 @@ In order to check the "readiness" of your CDN, you can optionally start the Read
 
 	sudo docker-compose -f docker-compose.readiness.yml up
 
+Integration Tests
+"""""""""""""""""
+
+There also exists TP and TO integration tests containers. Both of these containers assume that CDN in a Box is already running on the local system.
```
Comment (ocket8888) [inline docs/source/admin/quick_howto/ciab.rst:110]: s/exists/exist/

File: infrastructure/cdn-in-a-box/traffic_portal_integration_test/Dockerfile [hunk 2]
```diff
@@ -31,21 +31,13 @@ gpgkey=https://dl-ssl.google.com/linux/linux_signing_key.pub \n\
 " > /etc/yum.repos.d/google-chrome.repo
 
 RUN yum update -y
-
 RUN yum install -y java-1.8.0-openjdk git google-chrome-stable bind-utils net-tools
-
-RUN curl -LO https://nodejs.org/dist/v12.16.3/node-v12.16.3-linux-x64.tar.xz
```
Comment (ocket8888) [inline infrastructure/cdn-in-a-box/traffic_portal_integration_test/Dockerfile:37]: Doesn't getting rid of this cause the subsequent `npm install` steps to fail?

Comment (shamrickus) [reply infrastructure/cdn-in-a-box/traffic_portal_integration_test/Dockerfile:37]: `curl -LO` only downloads the file, there is nothing done with it (as far as I can tell) after the download.

Comment (ocket8888) [reply infrastructure/cdn-in-a-box/traffic_portal_integration_test/Dockerfile:37]: It's un-tarred into `/usr/local` on [line 37 of your branch](https://github.com/apache/trafficcontrol/pull/4682/files#diff-25b7184084d757a7bbf533c72127e049R37) - which means that as of now the integration container fails to actually build.

Comment (shamrickus) [reply infrastructure/cdn-in-a-box/traffic_portal_integration_test/Dockerfile:37]: That is interesting, it was definitely still working when I tested. Must have had something cached.

**Pull request merged**